### ISSUE TITLE
Check boxes to indicate availability/condition of sample occurrence records

### ISSUE THREAD

**Issue by kyule** [BioKIC/NEON-Biorepository#31]
**Is your feature request related to a problem? Please describe.**
In many cases, samples will be destroyed or otherwise altered during use by researchers. We do not want to remove destroyed samples from the portal as they still represent and will be associated with data. However, we want it to be clear that they are not available for future researchers. In other cases a sample will be altered (partially consumed, dissected, etc)

**Describe the solution you'd like**
Some type of check box for the occurrence record that indicates that the sample is or is not available for destroyed/lost/gifted samplles. Reason can be indicated in disposition

Some type of check box for the occurrence indicating that the sample has been altered. Specific types of alterations can be indicated in disposition.

**Comment by sunray1** [comment]
This is a part of a larger conversation on how to track sample usage in a better way, but doing this for now so the field is available.

To do:

- [x] Add field to SQL - assuming 'availability' is for this
- [x] Change default value
- [x] Backfill field according to disposition (2024-04-19)
- [x] Backfill field according to loan status
- [x] Add checkbox/dropdown to occurrence editor
- [x] Edit OccurrenceEditorManager so checkbox will update SQL
- [x] Edit loan module so will update based on loan status
- [x] Add to public display for viewing
- [x] Add to search page for filtering

**Comment by sunray1** [comment]
@kyule Could I get your help encoding the disposition to put into the availability field please? Could you take a look at Sheet 1 and try to fill out the gaps? I gave it a shot, just don't know what a good chunk of the text means. 0 = not available, 1 = available.

[disposition.xlsx](https://github.com/BioKIC/NEON-Biorepository/files/15028680/disposition.xlsx)

**Comment by kyule** [comment]
@sunray1 Here is a file -- little different than yours just in that I joined the collIDs and collectionNames. I think 99% of the ones that were confusing are from records pulled in from SEINet, etc. I vote that we keep all records in those collections as 1=available since anyone interested should go talk to the owner instituiton. 

[dispositions.csv](https://github.com/BioKIC/NEON-Biorepository/files/15031804/dispositions.csv)


We also need to consider material on loan as 0 = not available though. All material on loan for non-destructive use should be in the portal in a formal loan. Material that has gone out for destructive use may be in the portal as on loan (if the collection managers chose to do that), but may not be (I've argued against it as it's not really a loan).
